camel/messages/base.py

```diff
@@ -16,6 +16,8 @@
 from dataclasses import dataclass
 from typing import Any, Dict, List, Literal, Optional, Tuple, Union
 
+import imageio.v3 as iio
```

Comment:
Can we make this optional for now? IMO most users will not use video functionality for now. You can import this package before using it.